docs/application/native/tutorials/details/sec-privileges.md

```diff
@@ -49,7 +49,7 @@ Privileges](../../../tizen-studio/native-tools/api-checker.md).
 ## Mobile Native API Privileges
```

Comment:
Line 16: [tizen-manifest.xml](../process/setting-properties.md#setting-the-application-manifest) file.
Please use code tag for` tizen-manifest.xml` file name.